### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 573**
Add hint to set "experimental Nix feature 'nix-command'".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ With stable nix you can do:
 nix-build && ./result/bin/tex2nix
 ```
 
+(requires [experimental Nix feature 'nix-command'](https://nixos.wiki/wiki/Nix_command))
```
Comment (Mic92) [inline README.md:14]: I think we have a partial fix here: https://github.com/Mic92/tex2nix/blob/17cd0917ea39bc42a088f59490da0e72921ff075/tex2nix/__init__.py#L16

However maybe I should just fix all instances.

**Comment by Mic92** [comment]
Mhm. Actually that should fix all cases. Can you test with https://github.com/Mic92/tex2nix/commit/34f99e2fe5101c8ac480e8fbb9f2b5545c3a9aa0 ?

File: README.md (hunk 1 above)
Comment (573) [reply README.md:14]: This works without experimental-options as promised. Thanks !

**Comment by 573** [comment]
Confirming that [34f99e2](https://github.com/Mic92/tex2nix/commit/34f99e2fe5101c8ac480e8fbb9f2b5545c3a9aa0) works without experimental-options in nix.conf. Don't have to wait for nix 2.4 anymore ! Thanks so much !

**Pull request closed without merging**